docs/resources/domain.md

```diff
@@ -36,6 +36,8 @@ The following arguments are supported:
     the domain will accept email for sub-domains.
 * `dkim_key_size` - (Optional) The length of your domain’s generated DKIM key. Default value is `1024`.
 * `dkim_selector` - (Optional) The name of your DKIM selector if you want to specify it whereas MailGun will make it's own choice.
+* `click_tracking` - (Optional) (Enum: `yes`, `no` or `htmlonly`) The click tracking settings for the domain.
```

Comment:
Same as in previous PR: could you provide the defaults value for the params?

Reply:
Did you run the acc test? BTW: it could be great to have these new flags in the unit test as well 👍 .